package.json

```diff
@@ -1,136 +1,24 @@
 {
-  "name": "@pipecat-ai/react-native-daily-transport",
-  "version": "1.4.0",
-  "license": "BSD-2-Clause",
-  "description": "React Native library for connecting to RTVI client using Daily",
-  "source": "./src/index.tsx",
-  "main": "./lib/commonjs/index.js",
-  "module": "./lib/module/index.js",
-  "types": "./lib/typescript/module/src/index.d.ts",
-  "files": [
-    "lib",
-    "package.json",
-    "README.md"
+  "name": "pipecat-react-native-transports",
+  "private": true,
+  "workspaces": [
+    "transports/*",
+    "examples/*"
   ],
+  "packageManager": "yarn@3.6.1",
```

Comment:
👍

Reply:
I'd recommend adding two more things:
1. An `engines` field that specifies which version of nodejs to use e.g.
```
"engines": {
  "node": ">=24.0.0 <25.0.0",
  "yarn": ">=4.0.0"
}
```
2. An .nvmrc file in the root of the project with the recommended version of nodejs. E.g. an file with just one line `v24`.

If possible, I'd recommend setting `"type": "module"` too and use esmodules for everything. However, I can't remember if React Native ever ended up switching to esmodules so double check that first.